_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -62,7 +62,25 @@ changes to popular Bitcoin infrastructure software.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME:bitschmidty
+- **Lightning-based messenger application Juggernaut launches:**
+  In a [blog post][juggernaut blog] announcing the first release of Juggernaut,
+  John Cantrell describes how the messaging and wallet features are built using
+  [keysend payments][topic spontaneous payments].
+
+- **Lightning Loop using multipath payments:**
+  Lightning Labs' Lightning Loop product [now uses][lightning loop mpp blog]
+  [multipath payments][topic multipath payments] to convert onchain funds into
+  funds within LN channels.
+
+- **Blockstream Satellite 2.0 supports initial block download:**
+  Blockstream [outlines version 2.0 upgrades][blockstream satellite v2 blog] to
+  their satellite service which include expanded Asia-Pacific coverage and
+  additional bandwidth, enabling a full node to complete an initial sync
```

Comment:
_complete_ an initial sync _completely_ fees a bit repetitive. Consider ".. enabling a full node to do an initial sync using just the satellite feed."

Reply:
Tweaked. Thanks for the feedback.